src/haz3lcore/dynamics/Incon.re

```diff
@@ -1,6 +1,10 @@
 open Sets;
 
-let is_inconsistent_int = (xis: list(Constraint.t)): bool => {
+type b =
```

Comment:
A temporary type defined to simulate `bool`. If some value of `b` != `True`, that indicates that some constraint is consistent, and an additional `Constraint` is kept track of such that once it's satisfied, some dual is satisfied.

It's used to make the changes trackable, but it might be more intutive to change the signature of `is_exhaustive = (xi: Constraint.t): bool` to `satisfy = (xi: Constraint.t): option(Constraint.t)` and this auxillary type can be got rid of (drastic change!).